.github/workflows/d.yml

```diff
@@ -3,44 +3,58 @@ on: [push, pull_request]
 
 jobs:
   test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
     name: Dub Test
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
+        os: [ubuntu-20.04, windows-latest]
         dc:
           - dmd-2.096.1
           - dmd-2.095.0
           - ldc-1.26.0
           - ldc-1.20.0
+        arch:
+          - x86_64
 
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
 
-      - name: Install package dependencies
-        run: |
-          sudo apt-get update -y -qq
-          sudo apt-get install -y -qq libclang-10-dev
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3
+          bundler-cache: true
 
-      - uses: actions/setup-ruby@v1
+      # Ruby puts this on the path which breaks ldc
+      - name: Rename msys64/link.exe
+        if: startsWith(matrix.os,'windows')
+        shell: pwsh
+        run: mv C:\msys64\usr\bin\link.exe C:\msys64\usr\bin\msys-link.exe
 
-      - run: gem install bundler -v 1.17.3
+      - name: Setup D
+        uses: dlang-community/setup-dlang@v1
+        with:
+          compiler: ${{ matrix.dc }}
 
-      - run: bundle install
+      - name: Setup Visual Studio Command Prompt - Windows
+        if: startsWith(matrix.os,'windows')
+        uses: ilammy/msvc-dev-cmd@v1
 
-      - name: Install D compiler
-        uses: dlang-community/setup-dlang@v1.0.5
+      # NOTE use KyleMayes/install-llvm-action once https://github.com/KyleMayes/install-llvm-action/pull/11 is merged
+      - name: Setup LLVM
+        uses: aminya/install-llvm-action@defeault-directory
```

Comment:
The action itself is spelled incorrectly?

Reply:
Oops 😅 Fixed in https://github.com/atilaneves/dpp/pull/287/commits/037ab3963471c5446e5c1dbccc0c05bf5c42fe76